clang-tools-extra/docs/clang-tidy/checks/readability/use-span-first-last.rst

```diff
@@ -0,0 +1,24 @@
+.. title:: clang-tidy - readability-use-span-first-last
+
+readability-use-span-first-last
+===============================
+
+Checks for uses of ``std::span::subspan()`` that can be replaced with clearer
+``first()`` or ``last()`` member functions. These dedicated methods were added 
+to C++20 to provide more expressive alternatives to common subspan operations.
+
+Covered scenarios:
+
+========================== ====================
```

Comment:
Please enlarge first column to `s.subspan(s.size() - n)` (with all back-ticks) width. Second column is too wide.

Reply:
rst formatting is really tough 😬